exec/java-exec/src/main/java/org/apache/drill/exec/ssl/SSLConfigClient.java

```diff
@@ -94,7 +94,7 @@ private String getStringProperty(String name, String defaultValue) {
   private int getIntProperty(String name, int defaultValue) {
     int value = defaultValue;
     if ( (properties != null) && (properties.containsKey(name))) {
-      value = new Integer(properties.getProperty(name)).intValue();
+      value = Integer.valueOf(properties.getProperty(name)).intValue();
```

Comment:
```
    if (properties != null) {
      String property = properties.getProperty(name);
      if (property != null && property.length() > 0) {
        value = Integer.decode(property);
      }
```

Reply:
According to Java documentation, there are benefits in using valueOf for Double and Float.

https://docs.oracle.com/javase/7/docs/api/java/lang/Double.html#valueOf(double)
https://docs.oracle.com/javase/7/docs/api/java/lang/Float.html#valueOf(float)

Reply:
Do not always trust Java doc :). Here is the actual implementation:

```
    public static Double valueOf(double d) {
        return new Double(d);
    }
```